azure-cli/monitor/get_ApplicationInsights_ConnectionString.ps1

```diff
@@ -0,0 +1,21 @@
+function Get-ApplicationInsightsConnectionString {
+  param (
+    [Parameter(Mandatory = $true)]
+    [string]
```

Comment:
Use pascal case for parameter names - 
https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/strongly-encouraged-development-guidelines?view=powershell-7.2#use-pascal-case-for-parameter-names